### ISSUE TITLE
运行时出现如下问题

### ISSUE THREAD

**Issue by zzssyy** [jinfagang/tensorflow_poems#13]
ValueError: Attempt to reuse RNNCell <tensorflow.contrib.rnn.python.ops.core_rnn_cell_impl.BasicLSTMCell object at 0x7f8dde5d55f8> with a different variable scope than its first use.  First use of cell was with scope 'rnn/multi_rnn_cell/cell_0/basic_lstm_cell', this attempt is with scope 'rnn/multi_rnn_cell/cell_1/basic_lstm_cell'.  Please create a new instance of the cell if you would like it to use a different set of weights.  If before you were using: MultiRNNCell([BasicLSTMCell(...)] * num_layers), change to: MultiRNNCell([BasicLSTMCell(...) for _ in range(num_layers)]).  If before you were using the same cell instance as both the forward and reverse cell of a bidirectional RNN, simply create two instances (one for forward, one for reverse).  In May 2017, we will start transitioning this cell's behavior to use existing stored weights, if any, when it is called with scope=None (which can lead to silent model degradation, so this error will remain until then.)

**Comment by hitluobin** [comment]
the same problem occurs

**Comment by mingjun4503** [comment]
the same problem occurs

**Comment by jinfagang** [comment]
I think tensorflow update usage, later on I will update this repo

**Comment by molyswu** [comment]
the same problem occurs

**Comment by shenzhen2017** [comment]
I think is tensorflow1.1's bug

**Comment by 625673575** [comment]
@shenzhen2017 
yes, you got it ,It's a bug specifically happens on 1.1 version,after upgrade to tensorflow 1.2rc ,everything is ok

**Comment by molyswu** [comment]
Hi,
How to update for tensorfleo 1.2rc ,What is the command line?
Thanks!



sasunwu@126.com
 
发件人： 625673575
发送时间： 2017-05-31 15:35
收件人： jinfagang/tensorflow_poems
抄送： molyswu; Comment
主题： Re: [jinfagang/tensorflow_poems] 运行时出现如下问题 (#13)
@shenzhen2017
yes, you got it ,It's a bug specifically happens on 1.1 version,after upgrade to tensorflow 1.2rc ,everything is ok
